haskell/test/SMT/Model/BaseDAO/Proposal.hs

```diff
@@ -331,7 +328,12 @@ applyDropProposal mso proposalKey = do
     || proposalIsExpired
   then do
     unstakeProposerToken False proposal
-    removeFromProposalSortByLevel (proposal & plStartLevel) proposalKey
+    modifyStore $ \s ->
+      pure $ s
+        { sOngoingProposalsDlist = plistDelete proposalKey (s & sOngoingProposalsDlist)
+        }
+
+
```

Comment:
@rinn7e Redundant new lines.

Please clean up the changes a bit. I think there are redundant new lines in a number of places.

Reply:
@sras Removed the unneeded new lines, this should be good to go.